### PR TITLE
Warn user about secret starting from 0x00

### PR DESCRIPTION
**Pull request by szszszsz**
Tested on Ubuntu 16.04 with:
- Storage v0.43
- Pro v0.7

### CODE REVIEW

**Comment by szszszsz** [comment]
@jans23 Please provide translations for strings:
- `Nitrokey Pro v0.7 does not support secrets starting from null byte. Please change the secret.`
- `Nitrokey device does not support secret equal to null bytes. Please change the secret.`
If you would like to change them also for English please let me know.

**Comment by jans23** [comment]
Translation of the first string: "Nitrokey Pro v0.7 unterstützt keine Geheimnisse die mit Null Bytes beginnen. Bitte ändern Sie das Geheimnis."

@szszszsz "Nitrokey device does not support secret equal to null bytes." - I guess a better note would be "Your secret is null or empty. Please change the secret." Or am I wrong?

**Comment by szszszsz** [comment]
As for the second string I will provide the context - this is a response to hex secret consisting only of '00' (or base32 'AA's). On empty secret field only the slot name is transferred and the secret value is not changed on the device.

**Comment by jans23** [comment]
Yes. My point is that in such case of a "00" secret, its not a matter or Nitrokey limitation but its actually not a valid OTP secret per se. A secret of "00" doesn't make sense no matter which hardware is used. Right?

**Comment by szszszsz** [comment]
On reference implementation from [RFC page](https://tools.ietf.org/html/rfc4226#page-27) I cannot see any restrictions besides being 20 bytes (or 160 bits, its actually a recommendation) of binary data which will cover 20 null bytes too. My only concern is it could be chosen randomly by secret provider since it's a value like any other. If that would be the case then showing message that secret is null/empty might not suit well. But that is edge case and probably only one or two users will see this message ever (not counting the people wanting to break the device).

**Comment by jans23** [comment]
Perhaps the RFC isn't specific about it. But I still believe that a secret of "00" can hardly be considered a secret and therefore shouldn't be used. In such case I would suspect an error or other security issue. Are you ok with this string? - "Your secret is null or empty. Please change the secret."

**Comment by szszszsz** [comment]
Indeed, all nulls might indicate some implementation issue during secret creation as well. We could shorten this to just `Your secret is invalid. Please change the secret.'

**Comment by jans23** [comment]
"Your secret is invalid. Please change the secret." - "Ihr Geheimnis ist ungültig. Bitte ändern Sie das Geheimnis."

**Comment by szszszsz** [comment]
Translations updated.
